src/ztex/device_format.c

```diff
@@ -25,6 +25,12 @@
 // then it counts the device as not functioning one.
 #define DEVICE_TASK_TIMEOUT	5
 
+// Control the behavior in case where no devices are ready
+// because of errors. Startup behavior is not affected.
+// Set to 1 if you want to wait until any device is up.
+// Consider ztex_scan.h:ZTEX_SCAN_INTERVAL_DEFAULT
+const int WAIT_UNTIL_DEVICE_UP = 1;

```

Comment:
This is inconsistent use of #define vs. "const int", but regardless if we use "const int" we should also use the "static" specifier as otherwise the symbol is unnecessarily exported.